ui/helpers/utils/util.js

```diff
@@ -613,14 +610,6 @@ export function getSnapDerivationPathName(path, curve) {
 export const removeSnapIdPrefix = (snapId) =>
   snapId?.replace(getSnapPrefix(snapId), '');
 
-export const getSnapName = (snapId, subjectMetadata) => {
-  if (SNAPS_METADATA[snapId]?.name) {
```

Comment:
We can probably delete `SNAPS_METADATA` then right?

Reply:
I don't think we use it (for anything other than some example Snaps), so might as well remove it.

Reply:
Exactly!